stock_prices/stock_prices.py

```diff
@@ -2,14 +2,41 @@
 
 import argparse
 
+
+def findSmallest(x, y):
+    print(f"x, y: {x}, {y}")
+    if(x <= y):
+        return x
+    else:
+        return y
+
+
+def findBiggest(x, y):
+    print(f"biggest x, y: {x}, {y}")
+    if(x >= y):
+        return x
+    else:
+        return y
+
+
 def find_max_profit(prices):
-  pass
+    leastList = prices[0:-1]
+    leastPrice = min(leastList)
+    maxPrice = max(prices)
+    print(maxPrice - leastPrice)
+    return maxPrice - leastPrice
+
```

Comment:
Yea, I don't understand.  It failed that test and I kept moving.  I tend to fixate on problems and doing so, I've learned, doesn't really help.  Need "new scenery" after awhile.

Reply:
Another to way to explain the instructions of this problem is this way:

Given a list of integers, find the difference between integers _a_ & _b_ so that:
* The index of _a_ is less than the index of _b_
* The result of `_b_ - _a_` is maximized, i.e. the values of _a_ & _b_ are so that `_b_ - _a_` is the highest possible.

So for `[1050, 270, 1540, 3800, 2]`, the answer is `3,530` because _a_'s (`270`) index (1) is less than _b_'s (`3800`) index (3). Other values satisfy this (e.g. `1050, 3800`; `1540, 3800`), but `270, 3800` creates the largest `_b_ - _a_`, which is `3530`.